### PR TITLE
Steel bolt recipe

### PR DESCRIPTION
**Pull request by PropaneSoup**
Finally the simple additions thread gave me something actually simple ;)
http://smf.cataclysmdda.com/index.php?topic=5865.1710

### CODE REVIEW

File: data/json/recipes/recipe_ammo.json [hunk 1]
```diff
@@ -951,7 +951,39 @@
...
+      ["toolset", 7],
+      ["fire", -1],
+      ["forge", 7],
+      ["char_forge", 2],
+      ["oxy_torch", 2]
+      ]
+    ],
+     "components": [
+    [
+      ["steel_lump", 4]
```
Comment (KA101) [inline data/json/recipes/recipe_ammo.json]: Steel lumps are large amounts of steel--largest "scrap" amount available.  I'm thinking chunks might be more suitable.

Comment (PropaneSoup) [inline data/json/recipes/recipe_ammo.json]: I made it lumps because the recipe gives you ten whole bolts. Maybe 6 chunks then?

Comment (KA101) [inline data/json/recipes/recipe_ammo.json]: I'd have to look to be sure, but that's closer to reasonable.  Lumps, you're talking vehicle frames or anvils, not arrows, last I played.

Comment (PropaneSoup) [inline data/json/recipes/recipe_ammo.json]: Alright, hopefully I'll get to it tonight.

**Comment by KA101** [comment]
I've got a megamerge or two to make now that the freeze is off.  I can take care of it there.

**Pull request merged**